### ISSUE TITLE
"Error: <path> attribute d" when traces have one element and fill tozerox

### ISSUE THREAD

**Issue by sporto** [plotly/plotly.js#1912]
When traces have only one element and fill is set to tozerox.

I see this error:

```
Error: <path> attribute d: Unexpected end of attribute. Expected number, "…673599549.5,135L".
```

See fiddle here:

https://codepen.io/sporto/pen/eEYwmW?editors=1010

Thanks

**Comment by sporto** [comment]
Same error show if the trace has nulls

![image](https://user-images.githubusercontent.com/1005498/28550854-7486141a-7127-11e7-9f36-3e28079f953e.png)

**Comment by rreusser** [comment]
@sporto I submitted #1933 to fix the behavior when either the first or last point is isolate. There's a still a bit of interpretation regarding what the fill should look like when it encounters isolated points (should it ignore them? pass through them?) but at least this way it's consistent and doesn't give errors.

**Comment by rreusser** [comment]
@sporto #1933 has been merged and will be released mid-this-week with `1.30.0`, I believe. The error is gone and the fill now passes through single isolated points, which I think is as desired. Please let us know if you still have problems with this once it's released.

**Comment by sporto** [comment]
Thanks, I will
